src/istio/common/chart/templates/exemptions.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2024 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+
+apiVersion: uds.dev/v1alpha1
+kind: Exemption
+metadata:
+  name: istio
+  namespace: uds-policy-exemptions
+spec:
+  exemptions:
+    - policies:
+        - DisallowPrivileged
+        - RequireNonRootUser
+        - RestrictSELinuxType
+        - RestrictHostPathWrite
+        - RestrictVolumeTypes
+        - DisallowHostNamespaces
+        - DisallowSELinuxOptions
+        - DropAllCapabilities
+        - RestrictCapabilities
+        - RestrictHostPathWrite
+        - RestrictHostPorts
+        - RestrictProcMount
+        - RestrictSeccomp
+      matcher:
+        namespace: istio-system
+        name: "^.*"
+      title: "istio exemptions"
+      description: "Istio does some privileged things"
```

Comment:
This will still need some slimming down to specific pods/policies (https://github.com/defenseunicorns/uds-core/issues/1027).